src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: circle, area: " + (Math.PI * radius * radius) + " sq.units, radius: "
+                + radius + " units, color: " + new ColorSupplier().getRandomColor();
```

Comment:
Think about which variables should be local in the method and which should be class-level